packages/iris-grid/src/IrisGridTableModel.ts

```diff
@@ -196,7 +196,7 @@ class IrisGridTableModel
   updateFrozenColumns(columns: ColumnName[]): void {
     this.userFrozenColumns = columns;
     this.dispatchEvent(
-      new EventShimCustomEvent(IrisGridModel.EVENT.TABLE_CHANGED, {
+      new EventShimCustomEvent(IrisGridModel.EVENT.COLUMNS_CHANGED, {
         detail: this.table,
       })
```

Comment:
This isn't right - it should be emitting the table columns, not the table itself. 
```suggestion
      new EventShimCustomEvent(IrisGridModel.EVENT.COLUMNS_CHANGED, {
        detail: this.columns,
      })
```

Reply:
Updated, thanks!